Sources/StreamChat/Controllers/ListDatabaseObserver.swift

```diff
@@ -112,7 +112,7 @@ class ListDatabaseObserver<Item, DTO: NSManagedObject> {
     /// Used for observing the changes in the DB.
     private(set) var frc: NSFetchedResultsController<DTO>!
     
-    let itemCreator: (DTO) -> Item
+    let itemCreator: (DTO) -> Item?
```

Comment:
```suggestion
    let itemCreator: (DTO) -> Item
```
I don't think we need this to be optional. I would switch back to how it was